hack/dockerfiles/test.Dockerfile

```diff
@@ -16,15 +16,18 @@ RUN apk add --no-cache git libseccomp-dev make
 FROM gobuild-base AS buildkit-base
 WORKDIR /go/src/github.com/moby/buildkit
 COPY . .
+RUN mkdir .tmp; \
```

Comment:
update to my last comment: why create this directory at all?

Reply:
So that potentially we can create other files without contaminating the `buildkit` directory